platform/docs/docs/platform/services/data/StateSyncService.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 4
+sidebar_label: State Sync Service
+---
+
+# State Sync Service
+
+## Overview
+The state sync service is designed to allow short and long term memory of things such as
+annotations applied, last annotation state, hanging protocol viewport state,
+window level etc.  This allows for better interaction with things like navigation
+between hanging protocols, ensuring that the previously displayed layouts
+can be redisplayed after returning to a given hanging protocol.
+
+Currently, all the state sync service configurations have one of the following two
+lifetimes:
+
+* Application load - when the application is restarted, the state is lost
+* `clearOnModeExit` - which stores state until the mode onModeExit is called, and then throws away the remaining state.  This is useful for mode specific information.
+
+### Desired States
```

Comment:
By desired.... do you mean TODO? Is this something we usually put in API docs?

Also, should the header here be "Desired Stores" instead? Is it worth mentioning that the only current "store" is in memory?